olm-catalog/serverless-operator/generate_csv.sh

```diff
@@ -29,27 +29,21 @@ images=(
   ["eventing-webhook__eventing-webhook"]="${eventing}-webhook"
   ["storage-version-migration-eventing__migrate"]="${eventing}-storage-version-migration"
 
-  # the next three images have 0 replicas, they got removed, and point to 0.15 release images to migrate their deployments to 0
-  ["broker-controller__broker-controller"]="${registry}/knative-v0.15.2:knative-eventing-channel-broker"
-  ["broker-filter__filter"]="${registry}/knative-v0.15.2:knative-eventing-broker-filter"
-  ["broker-ingress__ingress"]="${registry}/knative-v0.15.2:knative-eventing-broker-ingress"
-
-  # the mt broker replaces the removed broker
   ["mt-broker-controller__mt-broker-controller"]="${eventing}-mtchannel-broker"
   ["mt-broker-filter__filter"]="${eventing}-mtbroker-filter"
   ["mt-broker-ingress__ingress"]="${eventing}-mtbroker-ingress"
   ["imc-controller__controller"]="${eventing}-channel-controller"
   ["imc-dispatcher__dispatcher"]="${eventing}-channel-dispatcher"
 
-  ["v0.16.0-broker-cleanup__brokers"]="${eventing}-broker-cleanup"
+  ["v0.17.0-pingsource-cleanup__pingsource"]="${eventing}-pingsource-cleanup"
   ["PING_IMAGE"]="${eventing}-ping"
   ["MT_PING_IMAGE"]="${eventing}-mtping"
   ["APISERVER_RA_IMAGE"]="${eventing}-apiserver-receive-adapter"
   ["BROKER_INGRESS_IMAGE"]="${eventing}-broker-ingress"
   ["BROKER_FILTER_IMAGE"]="${eventing}-broker-filter"
   ["DISPATCHER_IMAGE"]="${eventing}-channel-dispatcher"
 
-  ["KN_CLI_ARTIFACTS"]="${registry}/knative-v0.16.1:kn-cli-artifacts"
+  ["KN_CLI_ARTIFACTS"]="${registry}/knative-v0.17.1:kn-cli-artifacts"
```

Comment:
Just to doublecheck: Is this supposed to be 0.17.1?

Reply:
I looked here for the 0.17 bump:
https://github.com/openshift/release/blob/master/core-services/image-mirroring/knative/mapping_knative_v0_17_quay#L19

@navidshaikh or @rhuss  can you confirm ?

Reply:
+1 
v0.17.1 is the latest

heads up: We're planning another patch release v0.17.2 upstream (today/tomorrow) for some requirements from faas team, we'd make sure to update this once the CI/mirroring is in place

Reply:
cool! Thanks for confirming. If needed we can either update this PR, or just bump as needed on a separate PR

Reply:
kn v0.17.2 is ready, PTAL https://github.com/openshift-knative/serverless-operator/pull/544